README.md

```diff
@@ -22,11 +22,11 @@ This repository contains the XML source and schema files used to generate the au
 
 # How to contribute/participate
 We welcome participants and contributions! The SPDX License List is maintained by the SPDX Legal Team. Work and discussion is primarily done via:
-* **mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. Join the mailing list, see archive, and manage your subscription [via lists.spdx.org](https://lists.spdx.org/g/Spdx-legal). 
-* **bi-weekly calls**: We use conference calls to make decisions on topics and issues that may be difficult to only discuss via email or via Github. Calls are every other Thursday at noon Eastern Time (US). Conference call info is sent prior to the calls via the mailing list. If you join the mailing list, you should get a recurring invite at the beginning of each calendar year. Meeting minutes for the calls are in the [SPDX meetings repo](https://github.com/spdx/meetings/tree/master/legal); historical meeting minutes can be found at http://wiki.spdx.org/
-* **this Github repo**: comments and issues and PRs related to specific changes to the files that comprise the SPDX License List, e.g., new licenses, updates to an existing license, improving documentation, or other changes. 
+* **mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. To join the SPDX Legal Team mailing list, send an email to `Spdx-legal+subscribe@lists.spdx.org` or visit https://lists.spdx.org/g/Spdx-legal (where you can also see the list's archives).
+* **fortnightly calls**: We use fortnightly conference calls to make decisions on topics and issues that may be difficult to discuss only via email or GitHub. These calls are every other Thursday at 16:00 UTC. Information, including a link to join online, is sent prior to the calls to the SPDX Legal Team mailing list. Meeting minutes for the calls are in the [SPDX meetings repo](https://github.com/spdx/meetings/tree/master/legal); minutes from meetings before March 2020 can be found at http://wiki.spdx.org/
```

Comment:
I think the meeting time is US-based, not UTC-based. The UTC time changes when US enters/leaves Daylight Savings Time.

Reply:
Thank you for pointing this out to me! So it is not just us in Europe who have to cope with changing the clocks 😀 Speaking of time zones, it's a little late here, so I'll set this to draft status and work on it tomorrow.

Reply:
I've added another commit to the pull request to go back to using US Eastern Time. Thanks again!